src/components/Table/index.js

```diff
@@ -221,6 +222,16 @@ export default class Table extends Component {
         });
     }
 
+    detectActiveScroll() {
+        if (this.scrollableY.current) {
+            const { clientHeight, scrollHeight } = this.scrollableY.current;
+            const hasScroll = clientHeight < scrollHeight;
+            this.setState({ hasScroll });
+        } else {
+            this.setState({ hasScroll: false });
+        }
```

Comment:
let's avoid using state for this, instead make a method that simply returns a boolean